Source/WebCore/page/Navigation.h

```diff
@@ -185,6 +185,8 @@ class Navigation final : public RefCounted<Navigation>, public EventTarget, publ
 
     void updateNavigationEntry(Ref<HistoryItem>&&, ShouldCopyStateObjectFromCurrentEntry);
 
+    static Vector<Ref<HistoryItem>> filterHistoryItemsForNavigationAPI(Vector<Ref<HistoryItem>>&, HistoryItem&);
```

Comment:
Make up your mind, does it modify the Vector inline (in which case no return value) or does it return the modified vector (in which case the parameter should be `const`.

Reply:
Or, given that your implementation moves things out of the vector, likely we want `Vector<Ref<HistoryItem>>&&`.